docs/installation.rst

```diff
@@ -27,6 +27,18 @@ Setting up Vagrant
 
   It will typically take 5 to 30 minutes for the vagrant up to complete, depending on your network performance. If you experience any errors, see the :ref:`troubleshooting page <troubleshooting-vagrant>`.
 
+.. note:: You can get this error on Ubuntu and you can run `apt-get install nfs-common nfs-kernel-server` to fix it.
+
+
+.. code-block:: bash
+
+     It appears your machine doesn't support NFS, or there is not an
+     adapter to enable NFS on this machine for Vagrant. Please verify
+     that `nfsd` is installed on your machine, and try again. If you're
+     on Windows, NFS isn't supported. If the problem persists, please
+     contact Vagrant support.
```

Comment:
I think it would make more sense to add this to the prerequisites section: http://treeherder.readthedocs.io/installation.html#prerequisites